tfhe/src/shortint/server_key/mod.rs

```diff
@@ -55,6 +57,9 @@ impl Display for CheckError {
             Self::CarryFull => {
                 write!(f, "The carry buffer is full")
             }
+            Self::NoiseTooBig => {
```

Comment:
do we maybe want to store the noise level requested and the max noise level and format that in the error message ?

Same maybe for the carry full above, unsure

It's not necessary to go forward, could be nice potentially

Reply:
Done both